### DISCUSSION TITLE
How is Reinforcement Learning used in buildings?

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#95] [Key sector: Built Environment]
If there are any references to how RL is used in this space, I'd love to hear! TRIC researchers working on integration RL components into digital twins are interested in finding use cases for building demos of their methods

**Comment by esc-data-science** [comment]
We did have a speaker in our webinar series who did talk aout using RL in these types of applications [here](https://youtu.be/_Jc8ajUEFWQ?si=zuooOe4p-6wDcYhP)

**Comment by samyoung-dsci** [comment]
Energy Systems Catapult did a report on RL in energy a few months ago. There may be some useful references [there](https://es.catapult.org.uk/report/prospects-for-reinforcement-learning/)
